app/src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneService.java

```diff
@@ -29,16 +30,26 @@ public MarkerGeneService(MarkerGenesDao markerGenesDao, CellTypeSearchDao cellTy
         var ontologyLabelsCellTypeValues = cellTypeSearchDao.getInferredCellTypeOntologyLabels(experimentAccession, organismPart);
 
         var cellTypeMarkerGenes = ontologyLabelsCellTypeValues.isEmpty() ?
-                markerGenesDao.getCellTypeMarkerGenes(experimentAccession,
+                markerGenesDao.getCellTypeMarkerGenes(experimentAccession, CELL_GROUP_TYPE,
```

Comment:
@lingyun1010  I'm confused here - why we are passing "inferred cell type - ontology labels" group type for the authors label cell type fetch method. Can you clarify for me?

Reply:
This is a refactoring, `inferred cell type -ontology labels` was hard coded in DAO class. To reuse that DAO methods, for general `CELL_GROUP_TYPE`, I did the refactoring by moving the hard-coded value in the service class.